App/appReducer.js

```diff
@@ -12,6 +14,7 @@ const appReducer = combineReducers({
   [POSITIONS_NAMESPACE]: positionsReducer,
   [UI_NAMESPACE]: uiReducer,
   [FAVORITES_NAMESPACE]: favoritesReducer,
+  [SURVEY_NAMESPACE]: surveyReducer,
   [MATCH_NAMESPACE]: matchReducer,
```

Comment:
The match reducer already has survey questions and responses in it.  I think after you merge one of us should do a refactor to combine these